presto-orc/src/main/java/com/facebook/presto/orc/writer/DictionaryColumnWriter.java

```diff
@@ -413,6 +414,12 @@ public long getBufferedBytes()
         return getIndexBytes() + getDictionaryBytes();
     }
 
+    @VisibleForTesting
+    public long getRowGroupRetainedSizeInBytes()
```

Comment:
Does it make sense to annotate with VisibleForTesting? It is already public api.

Reply:
Yes, @VisibleForTesting is a marker or document annotation. It does nothing other than to give the readers indication that it is mainly for testing.